skeletons/OCTET_STRING.c

```diff
@@ -1748,7 +1748,7 @@ OCTET_STRING_free(const asn_TYPE_descriptor_t *td, void *sptr,
     case ASFM_FREE_UNDERLYING_AND_RESET:
         memset(sptr, 0,
                td->specifics
-                   ? ((const asn_OCTET_STRING_specifics_t *)(td->specifics))
+                   ? (size_t)((const asn_OCTET_STRING_specifics_t *)(td->specifics))
```

Comment:
This signedness mismatch better handled by a patch in a different place, here: https://github.com/vlm/asn1c/commit/19e12b5597e490bc9113f4fca2be9e2ea75dcdd5

Reply:
Ok, I'm going to drop this commit and force push